game/src/Game/LoadingScreen.gd

```diff
@@ -0,0 +1,32 @@
+extends Control
+
+@export var progress_bar: ProgressBar
+@export var quote_label: Label
+
+var loadthread: Thread
+var quotes: PackedStringArray = []
+
+func update_loading_screen(percent_complete: int, quote_should_change = false):
+	if percent_complete > 99:
+		get_tree().change_scene_to_file("res://src/Game/GameMenu.tscn")
```

Comment:
I think this should be moved to the end of `load_events`, rather than relying on actually hitting 100%, especially if the reported percentages become dynamic rather than fixed how they are now.